### DISCUSSION TITLE
Reduce size of published solution with IKVM

### DISCUSSION THREAD

**Discussion by michaelazarzar** [ikvmnet/ikvm#484] [Q&A]
Hi there,

I'm planning to use IKVM in order to communicate with Java application through RMI, and everything works perfect.
My `.csproj` contains `<PackageReference Include="IKVM" Version="8.7.5" />` and after publishing the solution,
the `.exe` size is ~100MB. 

I've tested it with `.jar` with only 1 method (i.e. `int add(int a, int b)`).

Is there any way to reduce the size of the outcome?

Thanks in advance.

**Comment by wasabii** [comment]
Really not a lot you can do about this today.
